### DISCUSSION TITLE
Custom app plugins showing 404

### DISCUSSION THREAD

**Discussion by aishyandapalli** [grafana/grafana#39530] [Plugins]
Hello team,

I am unable to load my custom plugins on grafana because it's not configure in routes.tsx. The code for plugin support is not implemented yet. May I know when is the ETA for this feature?

`// TODO[Router]
    // {
    //   path: '/plugins/:pluginId/page/:slug',
    //   templateUrl: 'public/app/features/plugins/partials/plugin_page.html',
    //   controller: 'AppPageCtrl',
    //   controllerAs: 'ctrl',
    // },`

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
